ex02.ex

```diff
@@ -41,7 +41,7 @@ defmodule Ex02 do
   # Do the same using the & syntax #
   ##################################
 
-  list2b = your_anonymous_function(1, 2)
+  list2b = &([&1 + &2, &1 - &2])
```

Comment:
No deduction, but I think it's easier to read without the extra parens:

~~~ elixir
list2b = &[&1 + &2, &1 - &2]
~~~